cpmpy/solvers/ortools.py

```diff
@@ -41,8 +41,7 @@
 from ..transformations.comparison import only_numexpr_equality
 
 class CPM_ortools(SolverInterface):
-    """
-    Interface to the python 'ortools' CP-SAT API
+    """Interface to the python 'ortools' CP-SAT API
```

Comment:
Why?

Reply:
Ah yes I tried everything to get the formatting right, there was something in the sfinx documentation about things being on the first line but that turned out to not matter. Reversed it